server/src/main/scala/org/http4s/server/package.scala

```diff
@@ -67,6 +67,12 @@ package object server {
   type AuthMiddleware[F[_], T] =
     Middleware[OptionT[F, ?], AuthedRequest[F, T], Response[F], Request[F], Response[F]]
 
+  /**
+    * An HTTP middleware that adds a context.
+    */
+  type ContextMiddleware[F[_], T] =
```

Comment:
Does anybody actually use these middleware aliases?  I've come to regard them as obfuscation more than anything.  I wonder what others think.

Reply:
We are using them on our project. I find them quite useful.

Reply:
We too use them.